DEVELOPERS.md

```diff
@@ -371,22 +369,6 @@ Some useful corrections are only made with `bundle exec rubocop -A`
 (upper case `A`) which applies "unsafe" corrections that may alter the
 semantics of the program.
 
-### Running the migrations
-
-Empty databases will cause Clover to crash.  You can run database
```

Comment:
This section should be reworded, not removed.  Developers should not be running `setup_database` for every migration, it's unnecessary to rerun all migrations.  test_up/dev_up should still be recommended for use after initial setup.

Reply:
I rewrote the commit to include an updated version of this (see above).